sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -31148,6 +31148,154 @@
                   "fieldSet": false,
                   "id": "efqfe2q"
                 },
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": false,
+                  "inputType": "text",
+                  "inputMask": "",
+                  "label": "Please estimate your weekly transportation costs.",
+                  "key": "studentDataAdditionalTransportCost",
+                  "placeholder": "",
+                  "prefix": "",
+                  "suffix": "",
+                  "defaultValue": "",
+                  "protected": false,
+                  "persistent": true,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "delimiter": true,
+                  "decimalLimit": 2,
+                  "requireDecimal": true,
+                  "validate": {
+                    "required": true,
+                    "multiple": "",
+                    "custom": ""
+                  },
+                  "conditional": {
+                    "show": "true",
+                    "when": "strongYouMayBeEligibleForAnAdditionalTransportationAllowanceIfThereIsNoLimitedLocalPublicTransitWhereYouLiveOrIfYouAreRequiredDueToSpecialCircumstancesDoYouWantToBeAssessedForAdditionalTransportationCostsStrong",
```

Comment:
Please change this long key to something shorter and check the impacted points (I don't think there will be great impacts).